hardware/index.md

```diff
@@ -219,15 +219,27 @@ Power to the micro:bit may be provided via 5V on the USB connector, or via a 3V
 
 The interface chip handles the USB connection, and is used for flashing new code to the micro:bit, sending and receiving serial data back and forth to your main computer.
 
+The V2.2X devices use two interface chip variants due to availability of the parts; 400K devices were built using nRF52833-QDAA and further devices are built using nRF52820-QDAA.
```

Comment:
Would it be possible to rephrase slightly? I had to read it twice as I almost misunderstood it as meaning that V2.2 has two interface chips.

Maybe something along the lines of `There are two variants of the V2.2X devices with a different  interface chip due to availability of the parts`, how does that sound?